udm-le.sh

```diff
@@ -96,18 +79,18 @@ initial)
 	fi
 
 	echo 'Attempting initial certificate generation'
-	${PODMAN_CMD} ${LEGO_ARGS} --accept-tos run && deploy_cert && add_captive && restart_unifi_os
+	${PODMAN_CMD} ${LEGO_ARGS} --accept-tos run && deploy_cert && add_captive && podman restart unifi-os &
```

Comment:
This will restart the container but the `unifi-os restart` also does a bunch of other things, the `start()` function in `/usr/sbin/unifi-os` checks mount points, among other things. Are we sure this is irrelevant? Maybe better `unifi-os restart &>/dev/null &`, just to be safe.

Reply:
Sorry for not proactively pushing up new commit with changes. I will test and push up by tomorrow.